tldr/semantic.py

```
@@ -167,6 +174,52 @@ def _confirm_download(model_key: str) -> bool:
         return False
 
 
+@contextlib.contextmanager
+def _suppress_hf_noise():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_suppress_hf_noise`

(ANN202)